src/Framework/Support/Facades/CurrencyFacade.php

```diff
@@ -76,6 +78,20 @@ public function formatToDecimal(Money $amount): string
      */
     public function formatToLocale(Money $amount, $locale = null): string
     {
+        $useAutoFormatting = give_get_option('auto_format_currency');
+        if (!class_exists(NumberFormatter::class) || !$useAutoFormatting) {
+            if ($useAutoFormatting) {
+                Log::warning(
+                    'Auto-formatting enabled but the INTL extension is not available. Please install the INTL extension to enable auto-formatting.'
```

Comment:
@Benunc Feel free to adjust this message, too.

Reply:
```suggestion
                    'Auto-formatting is enabled  at Donations > Settings > General > Currency but the INTL extension for PHP is not available. Please install the INTL extension to enable auto-formatting, or disable the Auto-formatting setting to prevent this error message. Most web hosts can help with installing and activating INTL. GiveWP is falling back to formatting based on the legacy settings.'
```